README.md

```diff
@@ -19,6 +19,7 @@ _mqttwarn_ supports a number of services (listed alphabetically below):
 * [dnsupdate](#dnsupdate)
 * [emoncms](#emoncms)
 * [execute](#execute)
+* [facebook messanger](#fbchat)
```

Comment:
Please correct to `facebook messenger` :-) In English it's a messAge but a messEnger